jbehave-support-core/src/main/java/org/jbehavesupport/core/report/XmlReporterFactory.java

```diff
@@ -157,12 +159,60 @@ private void copyResourceToReportDirectory(Resource resource) {
     private void copyResourceToReportDirectory(Resource resource, String targetFilePath) {
         File targetFile = new File(targetFilePath);
         try {
-            FileUtils.copyInputStreamToFile(resource.getInputStream(), targetFile);
+            String resourcePath = String.valueOf(resource.getURI());
+            String jarPath = resourcePath.replace(("!/" + resource.getFilename()), "").replace("jar:file:", "");
```

Comment:
Just a thought - is this detection of Jar by URI enough? Cant we get `jarFile.isFile() == true` in some other cases? (just wondering)